cfme/fixtures/service_fixtures.py

```diff
@@ -173,15 +177,17 @@ def order_service(appliance, provider, provisioning, dialog, catalog, request):
     assert provision_request.is_succeeded()
     if provision_request.exists():
         provision_request.wait_for_request()
-        if not BZ(1646333, forced_streams=['5.10']).blocks:
+        if not BZ(1775779, forced_streams=['5.10', '5.11']).blocks:
             provision_request.remove_request()
     yield catalog_item
     service = MyService(appliance, catalog_item.name)
     if service.exists:
         service.delete()
-    vm_name = '{}0001'.format(catalog_item.prov_data['catalog']['vm_name'])
-    vm = appliance.collections.infra_vms.instantiate(vm_name, provider)
-    vm.cleanup_on_provider()
+    name = catalog_item.prov_data['catalog']['vm_name']
+    for i in range(vm_count):
+        vm_name = f'{name}000'f'{i+1}'
```

Comment:
Is there a reason there are two f-strings here?

Reply:
That's an oversight. Fixed it.